open_flamingo/src/factory.py

```diff
@@ -66,8 +66,8 @@ def create_model_and_transforms(
         lang_encoder,
         text_tokenizer.encode("<|endofchunk|>")[-1],
         text_tokenizer.encode("<image>")[-1],
-        vis_dim=open_clip.get_model_config(clip_vision_encoder_path)["vision_cfg"][
```

Comment:
We should expose some flag to toggle between both options. Just want max flexibility.